packages/rocketchat-slackbridge/server/RocketAdapter.js

```diff
@@ -235,15 +235,21 @@ export default class RocketAdapter {
 	addChannel(slackChannelID, hasRetried = false) {
 		logger.rocket.debug('Adding Rocket.Chat channel from Slack', slackChannelID);
 		let slackResults = null;
-		let isGroup = false;
-		if (slackChannelID.charAt(0) === 'C') {
-			slackResults = HTTP.get('https://slack.com/api/channels.info', { params: { token: this.slackBridge.apiToken, channel: slackChannelID } });
-		} else if (slackChannelID.charAt(0) === 'G') {
-			slackResults = HTTP.get('https://slack.com/api/groups.info', { params: { token: this.slackBridge.apiToken, channel: slackChannelID } });
-			isGroup = true;
+		let slackMembers = null;
+
+		slackResults = HTTP.get('https://slack.com/api/conversations.info', { params: { token: this.slackBridge.apiToken, channel: slackChannelID } });
+		if (slackResults && slackResults.data && slackResults.data.ok === true) {
+			slackMembers = HTTP.get('https://slack.com/api/conversations.members', { params: { token: this.slackBridge.apiToken, channel: slackChannelID } });
```

Comment:
According to the documentation about this endpoint, when the "limit" parameter is not provided, the default limit is 100 items. How are you dealing with it, if the channel has more than 100 members?

Reply:
@MarcosSpessatto this is a good point.  It looks like the previous limitation was 1500 members rather than 100 members:

![2019-03-20T16 04 56-05 00](https://user-images.githubusercontent.com/227022/54719071-f6ed4d00-4b29-11e9-99b3-642b9fafa8b7.png)

The documentation at https://api.slack.com/methods/conversations.members does not state a hard limit, but recommends using a page size of 200.  A `for` loop using the `limit` and `cursor` parameters should be sufficient to make this work for any number of members.

Reply:
@MarcosSpessatto I do not have a test installation that would be appropriate to resolve this, and it's unlikely that I'll have time to look at it further.  Please feel free to update this PR.